accessible/clipboard.service.js

```diff
@@ -0,0 +1,98 @@
+var blocklyApp = blocklyApp || {};
```

Comment:
@NeilFraser: should I remove all the var blocklyApp = at the top of the files? I thought this was convention to make sure that even if files loaded in the wrong order or something everything wouldn't fall apart.